src/caremote.c

```diff
@@ -851,6 +868,14 @@ static int ca_remote_start(CaRemote *rr) {
                                 args[i++] = (char*) remote_casync;
                         }
 
+                        if (rr->rate_limit > 0) {
```

Comment:
here you use `rr->rate_limit > 0` as check, a few lines up you use `rr->rate_limit >= 0`...